api/check_post.py

```diff
@@ -2,10 +2,10 @@
 
 from api.utils import get_payload
 
-post_api = Blueprint('post', __name__)
+check_post_api = Blueprint('post', __name__)
 
 
-@post_api.route('/post', methods=['POST'])
+@check_post_api.route('/post', methods=['POST'])
```

Comment:
You must already use right endpoint names, for example '/apod' or '/picture' or 'wallpaper' or something that indicates what endpoint is for